background/state.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  version: null
```

Comment:
`State` sounds very general ; we might want to add some methods to it, I don't know... It was more a question than a comment though